tests/unit/test_utils_filesystem.py

```diff
@@ -59,3 +70,53 @@ def test_copy2_fixed_raises_appropriate_errors(create, error_type, tmpdir):
         copy2_fixed(src, dest)
 
     assert not dest.exists()
+
+
+def test_file_opener_no_file(process):
+    # FileOpener joins the subprocess even if the parent never sends the path
+    with FileOpener():
+        pass
```

Comment:
We can also assert that we have children in the `with` statement, otherwise this may pass incorrectly if we make a change that never starts the child process in the first place.